builtin/settingtypes.txt

```diff
@@ -2127,6 +2115,20 @@ show_technical_names (Show technical names) bool false
 #    Changing this setting requires a restart.
 enable_sound (Sound) bool true
 
+#    Unix timestamp (integer) of when the client last checked for an update
+#    Set this value to "disabled" to never check for updates.
+update_last_checked (Last update check) string
+
+#    Version number which was last seen during an update check.
+#
+#    Representation: MMMIIIPPP, where M=Major, I=Minor, P=Patch
+#    Ex: 5.5.0 is 005005000
+update_last_known (Last known version update) int 0
+
+#    This can be bound to a key to toggle camera smoothing when looking around.
+#    Useful for recording videos
+cinematic (Cinematic mode) bool false
```

Comment:
Would suit a little better directly below `continuous_forward`, as it's a tmp setting that one changes in-game via a key.

Reply:
It's already in that section, I haven't bothered sorting them in any particular way